### ISSUE TITLE
Right XML not well formatted 

### ISSUE THREAD

**Issue by gpessa** [Backbase/bb-cli#45]
If I try to use the command 
```bb export --structured```
i get an xml where the right file its wrong

```
<rights>
    <contextItemName>dbe</contextItemName>
    <name>link_1434108783198</name>
    <type>links</type>
    <itemRight>
      <name>link_1434108783198</name>
      <inherited>false</inherited>
    </itemRight>
    <itemRight>
      <0>A</0>
      <1>D</1>
      <2>M</2>
      <3>I</3>
      <4>N</4>
    </itemRight>
    <itemRight>
      <0>g</0>
      <1>r</1>
      <2>o</2>
      <3>u</3>
      <4>p</4>
      <5>_</5>
      <6>a</6>
      <7>d</7>
      <8>m</8>
      <9>i</9>
      <10>n</10>
    </itemRight>
  </rights>
```

and it should be something like :+1: 

```
<rights>
<itemRight name="link_1434108783198" inherited="true">
<securityProfile>ADMIN</securityProfile>
<sid>group_admin</sid>
</itemRight>
</rights>
```

**Comment by operatino** [comment]
I'm on it.

**Comment by operatino** [comment]
@igord fixed it in new, refactored export. Expect it to me merged in nearest week or so.

**Comment by operatino** [comment]
Here's a PR #46 already, hope to merge it soon.

**Comment by operatino** [comment]
Fixed in 0.2.0-beta, where we now have new export/import, with full API support, including content management.

0.2.0 stable will be released in few weeks, after some additional testing, to get it right now run

```
npm i bb-cli@0.2.0-beta -g
```
